app/src/main/res/values/strings.xml

```diff
@@ -173,4 +173,6 @@
     <string name="feedback_invalid_email">Input valid email address</string>
     <string name="drawer_subheader_games">Games</string>
     <string name="fork_me">Fork me</string>
+    <string name="arrowTagImage">Arrow Tagging Game</string>

```

Comment:
Never heard of something like that. I can add it though. I don't see any other strings with the `cd_` prefix.